tests/geo/test_surface_rzfourier.py

```diff
@@ -406,6 +406,17 @@ def test_from_pyQSC(self):
         np.testing.assert_allclose(s1.area(), s2.area())
         np.testing.assert_allclose(s1.volume(), s2.volume())
 
+        # test possible bug due to memory leak
+        from simsopt.configs import get_ncsx_data
+        _, _, ma = get_ncsx_data()
+        qsc = Qsc(ma.rc, np.insert(ma.zs, 0, 0), nfp=3, etabar=-0.408)
+        phis = np.linspace(0, 1/qsc.nfp, 2*ntor+1, endpoint=False)
+        thetas = np.linspace(0, 1, 2*mpol+1, endpoint=False)
+        full_torus = SurfaceRZFourier.from_pyQSC(qsc, r=0.1, ntheta=100, mpol=6, ntor=6)
+        full_period = SurfaceRZFourier(mpol=full_torus.mpol, ntor=full_torus.ntor, stellsym=full_torus.stellsym, nfp=full_torus.nfp, quadpoints_phi=phis, quadpoints_theta=thetas)
+        full_period.x = full_torus.x
+        # insert appropr
```

Comment:
Did you mean to write more here, or remove this line?

Reply:
Hi @landreman, I offered to write the unit test so Rogerio left it for me to input